src/styles/blocks/section.scss

```diff
@@ -0,0 +1,21 @@
+.main__section {
+  display: flex;
+  width: 1024px;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  box-sizing: border-box;
+}
+
+.section {
+  padding: 50px 41px;
+}
+
+.section__card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 15px 15px;
+  width: 198px;
```

Comment:
if you want to write 200px you just need to write at the start of file "normalize.scss" `* { box-sizing: border-box }`

Reply:
Unfortunately, this approach fails the test. That's why I duplicate this style